docs/rootfs-and-kernel-setup.md

```diff
@@ -119,6 +119,11 @@ Alpine Linux:
 
    # Then, copy the newly configured system to the rootfs image:
    for d in bin etc lib root sbin usr; do tar c "/$d" | tar x -C /my-rootfs; done
+   
+   # After this command, you can see these result
+   <img width="671" alt="Screen Shot 2021-03-02 at 21 19 49" src="https://user-images.githubusercontent.com/3442583/109695745-5456f900-7b9d-11eb-95b1-3720e1559292.png">
```

Comment:
I suggest to use a simple plain text message instead of a screenshot, something of the kind:
This command may trigger the following message: `tar: removing leading '/' from member names`. However, this is just a warning and it should be ok to continue with the setup flow.

Reply:
I changed my commit according to your suggestions